libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboard.tsx

```diff
@@ -0,0 +1,237 @@
+                    placeholder="Search"
+                  />
+                </Stack.Item>
+                <Stack.Item>
+                  <CommandBarButton
+                    className={classNames.filterButton}
+                    iconProps={{ iconName: "Filter" }}
+                    text={localization.InterpretVision.Dashboard.filter}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
+        <Stack.Item>
+          <Stack
+            horizontal
+            horizontalAlign="space-between"
+            verticalAlign="start"
+          >
+            <Stack.Item>
+              <Slider
+                max={80}
+                min={20}
+                className={classNames.slider}
+                label={localization.InterpretVision.Dashboard.thumbnailSize}
+                defaultValue={40}
+                showValue={false}
+                onChange={this.onSliderChange}
+              />
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
+        {this.state.selectedKey === VisionDatasetExplorerTabOptions.First && (
+          <Stack horizontal={false} grow tokens={{ childrenGap: "l1" }}>
+            <Stack
+              horizontal
+              tokens={{ childrenGap: "l1" }}
+              style={{
+                height:
+                  this.state.imageDim * 3 + Math.floor(this.state.imageDim / 2)
+              }}
+            >
+              <Stack
+                className={classNames.halfContainer}
+                tokens={{ childrenGap: "l1" }}
+              >
+                <Stack.Item>
+                  <TitleBar type={TitleBarOptions.Error} />
+                </Stack.Item>
+                <Stack.Item className={classNames.imageListContainer}>
+                  <ImageList
+                    imageDim={this.state.imageDim}
+                    openPanel={this.onButtonClick}
+                  />
+                </Stack.Item>
+              </Stack>
+              <Stack
+                className={classNames.halfContainer}
+                tokens={{ childrenGap: "l1" }}
+              >
+                <Stack.Item>
+                  <TitleBar type={TitleBarOptions.Success} />
+                </Stack.Item>
+                <Stack.Item className={classNames.imageListContainer}>
+                  <ImageList
+                    imageDim={this.state.imageDim}
+                    openPanel={this.onButtonClick}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack>
+          </Stack>
+        )}
+      </Stack>
+    );
+  }
+
+  private onButtonClick = () => {
+    this.setState({ panelOpen: !this.state.panelOpen });
+  };
+
+  private onSliderChange = (value: number) => {
+    if (this.state.selectedKey === VisionDatasetExplorerTabOptions.First) {
+      this.setState({ imageDim: Math.floor((value / 100) * 400) });
```

Comment:
can you add comment on how are these numbers 400 and 100 selected? 
Comments in any place where calculations are made is appreciated, thanks.